lib/js/components/Modals/Modal/Modal.vue

```diff
@@ -0,0 +1,417 @@
+<template>
+	<div
+		class="modal"
```

Comment:
did you verify if this class name won't conflict with bulma modal?

Reply:
@kvas-damian do You have any suggestions? `modalMain`? Other variants will be similar: `modalDialog`

Reply:
Didn't think about it. Maybe some prefix like `ds` or `b` like Bethink?

Reply:
I used `ds`.